integration/ct_integration_test.go

```diff
@@ -124,3 +164,28 @@ func certsFromPEM(data []byte) []ct.ASN1Cert {
 	}
 	return chain
 }
+
+func getChain(path string) ([]ct.ASN1Cert, error) {
+	certdata, err := ioutil.ReadFile(filepath.Join(*testdata, path))
+	if err != nil {
+		return nil, fmt.Errorf("Failed to load certificate: %v", err)
+	}
+	return certsFromPEM(certdata), nil
+}
+
+func awaitTreeSize(ctx context.Context, logClient *client.LogClient, size uint64, exact bool) (*ct.SignedTreeHead, error) {
```

Comment:
Possibly true and I do check it in the backend integration test. It depends if you're worried about bugs that could return an incorrect tree size.